subprojects/packagefiles/pcre2/meson.build

```diff
@@ -147,10 +147,16 @@ pkg.generate(libraries : pcre2_32_lib,
 #### tests
 
 if not meson.is_cross_build() # wine wrappers are not bat-friendly, this would need better testing
+  link_args = []
+  if host_machine.system() == 'windows'
+    link_args += ['/STACK:2500000']
```

Comment:
This will technically work, even though it sets `link_args = [ ['/STACK:2500000'] ]`, because it will be automatically flattened to a one-dimensional array in the process of being consumed by `executable()`. However, it's unnecessary. :D

Reply:
was not aware of this, I don't really use meson in the past this seemed like the right behavior.